src/store/reducers/alcohol.js

```diff
@@ -1,5 +1,29 @@
-import { initialState } from '../data/alcohol';
+import * as actions from '../actions/actionTypes';
 
-const alcoholReducer = (state = initialState, action) => state;
+const setLevels = (state, action) => {
+  
+  const results = action.drinks.map((type, key) => {
+    let id = key;
+    let name = type.strAlcoholic;
+    let typeInfo = {
+      id: id,
+      name: name
```

Comment:
when the key and the variable assigned to it have the same name there is no need to duplicate them so, this can be
 {id, name}

Reply:
or if you prefer you can avoid the creation of the 2 variables above and just calling them inside the object as you prefer